ux/musicplayer.java

```diff
@@ -0,0 +1,13 @@
+/*
+*************************************************************************************************************************
+**** This Java class will handle playing the drums. ****
+
+	What kind of methods does it need?
+	- play_drums
+		- it could have some kind of global variable that will increase or decrease the volume and/or speed. 
+		  Maybe one for each? volumen and speed?
+		- Will it play drums for a certain amount of time? Will it play drums until it gets some other input?
+	- change_volume
+	- change_speed
+*************************************************************************************************************************
+*/
```

Comment:
I'm thinking of having the drums either respond to a green house gas calculation and also play at random intervals.  Intensity is determined by the greeenhouse calculator, or is determined randomly.

Reply:
I was thinking there will be some other object that "decides" what it should tell the output objects to do based on the greenhouse calculator. Or I guess the output objects could keep checking to see when the calculation changes and then change their behavior accordingly.